### DISCUSSION TITLE
CTklabel has no attribute “borderwidth”

### DISCUSSION THREAD

**Discussion by maxwangwyf** [TomSchimansky/CustomTkinter#2551] [Q&A]
why

**Comment by dipeshSam** [comment]
Good question. While supporting corners in `CTkLabel`, it is still borders are not implemented. In upcoming updates, @TomSchimansky may take a look over this matter.

There is only `CTkFrame` which has borders. And, `CTkScrollableFrame` is written over `CTkFrame`. You can write label in same manner.

Regards.
